Source/Time/DateTimeOffsetExtensions.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+
+namespace Dolittle.Time
+{
+    /// <summary>
+    /// Extensions for DateTime
+    /// </summary>
+    public static class DateTimeOffsetExtensions
+    {
+        /// <summary>
+        /// Performs a lossy equals algorithm
+        /// </summary>
+        /// <param name="first"></param>
+        /// <param name="second"></param>
+        /// <returns></returns>
+        public static bool LossyEquals(this DateTimeOffset first, DateTimeOffset second)
```

Comment:
It would probably be better to do this with a DateTimeComparer in case we need to capture <>, <= =>.

Reply:
Absolutely, and we MUST have this in the future. DateTime stuff will always be a pain point